package.json

```diff
@@ -72,18 +72,18 @@
     "mocha": "^10.0.0",
     "nock": "^13.0.2",
     "nyc": "^15.1.0",
+    "oclif": "^3",
     "proxyquire": "^2.1.3",
     "semantic-release": "^19.0.3",
     "sinon": "^10.0.0",
     "tildify": "^2.0.0",
     "tmp": "^0.2.1",
     "ts-node": "^10.9.1",
     "tslib": "^2.3.1",
-    "typescript": "^4.4.3",
-    "oclif": "^3"
+    "typescript": "^4.4.3"
   },
   "optionalDependencies": {
-    "ngrok": "^4.3.3"
+    "ngrok": "^5.0.0-beta.1"
```

Comment:
@brleinad Thanks for raising the PR, We don't want to update ngrok dependency to beta version. We will wait until ngrok release it as GA version. Meanwhile we will add this issue to our backlog and will look into this.

Reply:
Sounds good. I'll close this PR then